x-pack/plugins/secops/public/components/error_toast/index.tsx

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiGlobalToastList, Toast } from '@elastic/eui';
+import { noop } from 'lodash';
+import React from 'react';
+import uuid from 'uuid';
+
+interface Props {
+  toastLifeTimeMs?: number;
+}
+
+interface State {
+  toasts: Toast[];
+}
+
+let showErrorHandler = ({ id = uuid.v4(), title, message }: ShowErrorParams): void => noop();
+export const showError = ({ id = uuid.v4(), title, message }: ShowErrorParams) =>
```

Comment:
Just curious, but is there a cleaner way to manage default params with regards to proxy functions like these? Does `id = uuid.v4()` have to be duplicated between `showErrorHandler()`, `showError()` and `ErrorToast.showError()`?

Reply:
I can remove all but the public top level one probably unless I swap it out for redux/redux observable.